fission-cli/library/Fission/CLI/Connected.hs

```diff
@@ -126,7 +126,7 @@ mkConnected ::
 mkConnected inCfg ipfsTimeout = do
   attempt (Key.Store.fetch $ Proxy @SigningKey) >>= \case
     Left _err -> do
-      CLI.Error.put NoKeyFile "Cannot find key. Please run: fission user register"
+      CLI.Error.put NoKeyFile "Cannot find key. Please run: fission user login"
```

Comment:
`login` won't create a user if this is their first account. They will need to run `setup` unless they know that they have an existing account.